boards/arc/emsdp/Kconfig.defconfig

```diff
@@ -24,10 +24,7 @@ config SPI_DW
 if SPI_DW
 
 config SPI_DW_FIFO_DEPTH
-	default 32
-
-config SPI_DW_ARC_AUX_REGS
-	default n
+	default 16
```

Comment:
I'm wondering about why do we change fifo size for emsdp?

Was it incorrect? Or DW and DFSS peripherals have different FIFO size on emsdp?

Reply:
emsdp fifo depth is 32 for DW. 16 for DFSS.

Reply:
IMHO that needs to be moved to the .dts as the right place for HW description. That will code simpler and .dts more informative (more complete).